eZ/Publish/Core/Search/Common/FieldValueMapper/FloatMapper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/pull/1697/commits/4f7f55da0bc1367cf468bbbe90d8cabc37b48ae1.